# Action goal : client sends goal of a end position
float64 spiral_constant
float64 angular_velocity
float64 radius
---
# Action result
bool spiral_complete
---
# Action Feedback
float64 current_radius
